README.md

```diff
@@ -73,6 +73,13 @@ or on Arch with
 sudo pacman -S python-pipx
 ```
 
+You will also need the USB development libraries; on Ubuntu/Debian, install
+with
+
+```bash
+sudo apt-get  --no-install-recommends install libusb1-dev
```

Comment:
Why is `libusb1-dev` necessary? What error message do you get without it? Would `libusb-1.0-0` suffice?

Reply:
The error was that "No supported devices found."  That is present even with the library but without the -dev package.  It appears that one of the vendored libraries tries to compile something against it or something.

Reply:
Actually, you are correct.  I made a mistake with my testing.  I will fix this.

Reply:
Great, thanks so much for tracking this down!

Would you be able to explicitly mention the "No supported devices found" error? Something like,

> If your printer is connected but you receive the error `No supported devices found` then please ensure `libusb` has been installed. On Ubuntu/Debian it can be installed with

Reply:
Sorry @jgoerzen, did you see my above comment?

Reply:
Resolved in #77.